back/spec/acceptance/projects_spec.rb

```diff
@@ -631,6 +631,22 @@
         expect(json_response).to eq({ data: { totalSubmissions: 3 } })
       end
     end
+
+    delete 'web_api/v1/projects/:id/inputs' do
+      let(:project) { create :continuous_project }
+      let(:id) { project.id }
+
+      example 'Delete all inputs of a project' do
+        create_list :idea, 2, project: project
+        create :idea
+
+        do_request
+        expect(response_status).to eq 200
+        expect(Project.find(id)).to eq project
+        expect(project.reload.ideas_count).to eq 0
+        expect(Idea.count).to eq 1
+      end
+    end
```

Comment:
Please add a spec that describes what happens when deleting is not allowed, otherwise we do not have proof that the policy class is used correctly.